src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,6 +62,23 @@ private void configureButtonBindings() {
     
   }
 
+  /**
+   * set the drive train type
+   * @param type
+   */
+  public void setDriveTrainType(int type) {
+    SmartDashboard.putNumber(Constants.DRIVE_TRAIN_TYPE, type);
```

Comment:
Wouldn't this be better as a String with the enum name()?

Reply:
At the time when I wrote this, I did not consider it. Yes, it will probably makes more sense to use string. Wasn't sure Java support string enum.